data/db-router.js

```diff
@@ -0,0 +1,98 @@
+const router = require('express').Router();
+
+const db = require('./db.js');
+
+router.post('/', (req, res) => {
+  const newPost = req.body;
+  if (newPost.title && newPost.contents) {
+    db.insert(newPost)
+      .then(post => {
+        res.status(201).json(newPost);
```

Comment:
be careful here, you're only returning specifically what you passed in but it will not contain the id, createdat, or updatedat properties that everything in the database should contain. this should call for a findById method here to pull the new item from the database based on the `id` being sent back in this .then block (what you referenced as `post`)